### PR TITLE
Upgrade jackson 2.17.0

### PR DESCRIPTION
**Pull request by tung-vu-td**
In this PR:
- Update a list of test dependencies.

Should have no impact on production

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -4,15 +4,15 @@ VERSION_NAME=3.0.0
...
+POM_ORG_URL=https://www.treasuredata.com/
 POM_SCM_URL=https://github.com/treasure-data/KeenClient-java.git
 POM_SCM_CONNECTION=scm:git@github.com:treasure-data/KeenClient-java.git
 POM_SCM_DEV_CONNECTION=scm:git@github.com:treasure-data/KeenClient-java.git
 POM_LICENSE_NAME=The Apache Software License, Version 2.0
 POM_LICENSE_URL=http://www.apache.org/licenses/LICENSE-2.0.txt
 POM_LICENSE_DIST=repo
-POM_DEVELOPER_ID=keenlabs
-POM_DEVELOPER_NAME=Keen IO
+POM_DEVELOPER_ID=treasuredata
```
Comment (kietdo360) [inline gradle.properties]: Is it ok to change to TD while we are forking this from Keen Lab ?

**Review by kietdo360 — APPROVED** [review]
LGTM with one minor comment

**Review by nmpennypacker — APPROVED** [review]
Reviewed for data exfiltrations

File: gradle.properties (hunk 1 above)
Comment (tung-vu-td) [reply gradle.properties]: Nice catch, we should list both developers. I fixed this in [98ddad7](https://github.com/treasure-data/KeenClient-Java/pull/13/commits/98ddad766f3a5ba5ee8a613c75111ca7293bfe21)

**Comment by kietdo360** [comment]
Still LGTM

**Pull request merged**